infrastructure/src/main/java/org/corfudb/infrastructure/log/SegmentHandle.java

```diff
@@ -52,18 +51,14 @@ public synchronized void release() {
     }
 
     public void close() {
-        Set<FileChannel> channels = new HashSet<>(
-                Arrays.asList(writeChannel, readChannel)
-        );
-
-        for (FileChannel channel : channels) {
+        Arrays.asList(writeChannel, readChannel).forEach(channel -> {
             try {
                 channel.force(true);
             } catch (IOException e) {
-                log.debug("Can't force updates in the channel", e.getMessage());
+                log.debug("Can't force updates in the channel: {}", e.getMessage());
```

Comment:
Probably should be a warn and not a debug message. Can you also log the segment number.

Reply:
It was `debug` all the time if you'd like to change it, we would need another PR